migrations/versions/72936c3fdac6_add_associated_user_roles_tables_and_.py

```diff
@@ -0,0 +1,56 @@
+"""add associated user roles tables and data and update project has associated user
+
+Revision ID: 72936c3fdac6
+Revises: 09a38764b3a0
+Create Date: 2023-10-05 12:58:29.304829
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from userportaldatamodel.models import AssociatedUserRoles
+from sqlalchemy.orm import Session
+
+# revision identifiers, used by Alembic.
+revision = '72936c3fdac6'
+down_revision = '09a38764b3a0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+
+    op.add_column(
+        "project_has_associated_user",
+        sa.Column("active", sa.Boolean(), nullable=False, server_default='true')
+    )
```

Comment:
When a column is created like this is it populating with the default value 'true' also older records or just the ones that are created after. Could you double check this. And if the old ones are not populated with `true` we should add it to this migration.

Reply:
I test it and server_default='true' backfills all existing columns and sets them to True.

Reply:
perfect